.github/workflows/e2etests.yml

```diff
@@ -8,25 +8,53 @@ jobs:
     strategy:
       matrix:
         node: ["16"]
-        browser: ["firefox", "chrome"]
+        browser: ["chrome"]
+        # browser: ["firefox", "chrome"]
+    
+    # https://docs.github.com/en/actions/using-containerized-services/about-service-containers
+    services:
+      postgres:
+        image: postgres:alpine
+        env:
+          POSTGRES_PASSWORD: pass
+        ports:
+          - 5432:5432
+        volumes:
+          - ${{ github.workspace }}/init-project-db.sh:/docker-entrypoint-initdb.d/init-user-db.sh
+      keystone-swift:
+        image: ghcr.io/cscfi/keystone-swift:latest
```

Comment:
this repo, where is it?
there is no image here: https://github.com/CSCfi/docker-keystone-swift/packages, maybe you need to add https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages to that repository then you will be able to pull it with `ghcr.io/CSCfi/docker-keystone-swift`